Makefile

```
@@ -20,10 +20,14 @@ CI_FLAGS =\
 	DBT_LOG_FORMAT=json
 
 .PHONY: dev
```

I didn't notice this the first time around!
```suggestion
.PHONY: dev_req
```